security-privacy-questionaire.md

```diff
@@ -0,0 +1,97 @@
+# WebCodecs - Security and Privacy Questionnaire
+
+This document answers the [W3C Security and Privacy
+Questionnaire](https://www.w3.org/TR/security-privacy-questionnaire/) for the
+WebCodecs specification.
+
+Last Update: 2019-10-18
+
+**2.1. What information might this feature expose to Web sites or other parties, and for what purposes is that exposure necessary?**
+
+This feature may expose additional information about what media a user agent is
+capable of encoding or decoding.  Such information is largely already available
+through other media APIs such as MediaRecorder, Media Source Extensions, WebRTC,
+and Media Capabilities, but lower-level media APIs, such as this one may subtly
+expose more information or make it easier to obtain the same information.  For
+example, this feature may expose timing information about how long media takes
+to encode and/or decode.
+
+This is a necessary part of exposing a lower-level API that provides more direct
+control.
+
+
+**2.2. Is this specification exposing the minimum amount of information necessary to power the feature?**
+
+Yes.  It will only expose information when it is necessary to expose controls.
+For example, if a necessary control is added, then it would be easy to derive
+whether the user agent is capable of supporting that control by attempting to
+use it and seeing if it fails.
+
+**2.3. How does this specification deal with personal information or personally-identifiable information or information derived thereof?**
+
+There is no PII involved.
```

Comment:
PII could be the media that flows through WebCodecs, but I'd say that's out of scope and a concern of the MediaStreams spec. This could change if opaque streams were introduced, since you may be able to gather timing info from the encoder/decode operations.